srcpkgs/nextdns/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'nextdns'
+pkgname=nextdns
+version=1.4.35
+revision=1
+build_style=go
+go_import_path=github.com/nextdns/nextdns
+short_desc="NextDNS CLI client (DoH Proxy)"
+maintainer="Giuseppe Fierro <gspe@ae-design.ws>"
+license="MIT"
+homepage="https://nextdns.io/"
+distfiles="https://github.com/${pkgname}/${pkgname}/archive/v${version}.tar.gz"
+checksum=7a7c0829ca52a711f6fc813e6d913577670c95ddd8b3ab5d17da2b69440a3edc
+conf_files="/etc/nextdns/nextdns_conf"
```

Comment:
is there a reason this is not .conf? Does upstream prefer _conf ?

Reply:
Good catch. The upstream accepts arbitrary file names.

Reply:
The upstream config file is created with the `nextdns` commands and stored internally somewhere, so I don't know its name exactly